packages/pagination/src/elements/OffsetPagination/components/Gap.tsx

```diff
@@ -6,13 +6,23 @@
  */
 
 import React, { forwardRef, LiHTMLAttributes } from 'react';
-import { StyledGap } from '../../../styled';
+import { StyledGapListItem } from '../../../styled';
+import { useText } from '@zendeskgarden/react-theming';
 
-const GapComponent = forwardRef<HTMLLIElement, LiHTMLAttributes<HTMLLIElement>>((props, ref) => (
-  <StyledGap {...props} ref={ref}>
-    …
-  </StyledGap>
-));
+const GapComponent = forwardRef<HTMLLIElement, LiHTMLAttributes<HTMLLIElement>>((props, ref) => {
+  const ariaLabel = useText(
+    GapComponent,
+    props,
+    'aria-label',
+    'Ellipsis indicating non-visible pages'
```

Comment:
Is it worth passing a `count` (naming tbd) prop to the gap component so that this default label can be customized with "Ellipsis indicating ${count} non-visible pages"? Certainly non-essential; perhaps nice to have.

Reply:
I like that idea. Looks like the calculation is all there already, too.

Reply:
Actually, the calculation is a bit more involved than I thought. Definitely something to consider in the future though.

Reply:
Ok, I'm totally fine to defer. But we'll probably need to punt to a future major, because we'd want the `labels` API to follow suit with a function.